src/sfdt/__tests__/toggleBookmark.test.js

```diff
@@ -114,12 +114,12 @@ describe('toggleBookmark', function() {
 			expect(firstInline[2].hasFieldEnd).toBe(true);
 			expect(lastInline[1].fieldType).toBe(1);
 
-			const newSfdt = JSON.parse(JSON.stringify(bookmarkStartEndingInDifferentInline));
-			const toggleOff = toggleBookmark(
-				newSfdt,
-				'COND::9e7d0dc1-b9ed-4baa-9399-a4c4c9be96d4',
-				true
-			);
+			// const newSfdt = JSON.parse(JSON.stringify(bookmarkStartEndingInDifferentInline));
```

Comment:
on purpose ?

Reply:
These part is not used, but we had condition testing for them before and this test has no toggleBookmark implementation either
Since I don't know what is supposed to go there, I'll leave it as it is and we shall update later